sdk/tables/azure-data-tables/samples/tables_entity_operations.cpp

```diff
@@ -12,26 +13,22 @@ using namespace Azure::Data::Tables;
 using namespace Azure::Data::Tables::Models;
 const std::string TableName = "table";
 
-std::string GetConnectionString()
-{
-  const static std::string ConnectionString = "";
-
-  if (!ConnectionString.empty())
-  {
-    return ConnectionString;
-  }
-  const static std::string envConnectionString = std::getenv("STANDARD_STORAGE_CONNECTION_STRING");
-  if (!envConnectionString.empty())
-  {
-    return envConnectionString;
-  }
-  throw std::runtime_error("Cannot find connection string.");
-}
-
 int main()
 {
-  auto tableServiceClient = TableServiceClient::CreateFromConnectionString(GetConnectionString());
-  auto tableClient = TableClient::CreateFromConnectionString(GetConnectionString(), TableName);
+  const std::string accountName{std::getenv("ACCOUNT_NAME")};
+  auto credential = std::make_shared<Azure::Identity::ChainedTokenCredential>(
+      Azure::Identity::ChainedTokenCredential::Sources{
```

Comment:
Isn't pipeline credential already in DefaultAzureCredential?

Or is there something specific here?

Reply:
It isn't. We need to come up with a resource deployment solution for our CI pipelines that works with the existing DAC.